bedrock/firefox/templates/firefox/features/adblocker.html

```diff
@@ -24,133 +24,153 @@
+      On Firefox, you can use <a href="{{ privacy }}">Privacy</a> or <a href="{{ blocking }}">Content Blocking</a> settings to get even more control over ad trackers that serve you the ads.
+      {% endtrans %}
+    </p>
+  </section>
+
+  <section class="mzp-l-content l-content-image">
+    {{ high_res_img('firefox/features/adblocker/content-blocking-title.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
   </section>
 
   <section class="mzp-l-content l-content-narrow">
-    <h2 class="section-title">{{ _('Why use ad blockers?') }}</h2>
-    <p>{{ _('Ad blockers can also be used to make the web faster.') }}</p>
-    <p>{{ _('Ads can slow down sites while they load, or get in the way of a streamlined browsing experience.') }}</p>
-    <p>{{ _('Ads can also be flashy and annoying, distracting you from the stuff you really want to see.') }}</p>
-    <blockquote>{{ _('“Ads can slow down sites… or get in the way of a streamlined browsing experience.”') }}</blockquote>
+    <h2 class="section-title">{{_('Choose your level of protection') }}</h2>
     <p>
-    {% trans %}
-    Finally, ad companies can track your movements online. It’s called "re-targeting" and it works like this: You visit a site and click on a product. That site puts a small piece of code in your browser called a cookie. When you surf the web, sites will know you clicked on that product and they’ll display ads for it.
-    {% endtrans %}
+      {% trans %}
+      To start, click on the Firefox menu in the top right-hand corner of your screen. It looks like four lines stacked on top of each other. In the drop-down menu, click on Content Blocking. You should see a blue pop-up with different selections.
+      {% endtrans %}
     </p>
   </section>
 
   <section class="mzp-l-content l-content-narrow">
-    <h2 class="section-title">{{ _('Why you might not use an ad blocker') }}</h2>
+    <h2 class="section-title">{{_('Go easy with Standard mode') }}</h2>
     <p>
-      {% trans %}
-      Many media and news organizations make most of their money from advertising. If you block ads on their sites, they won’t get paid. If enough people block ads, business could suffer, and so will the quality of reporting and writing. Luckily, most ad blockers let you make exceptions for your favorite sites.
+      {% trans url=url('firefox.features.private-browsing') %}
+      If ads don’t bother you and you don’t mind being followed by trackers and third-party cookies, then the Standard setting should work for you. To get trackers off your tail in Standard mode, use a <a href="{{ url }}">Private Browsing</a> window.
       {% endtrans %}
     </p>
   </section>
 
   <section class="mzp-l-content l-content-image">
-    {{ high_res_img('firefox/features/privacy-illustration2.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
+    {{ high_res_img('firefox/features/adblocker/content-blocking.png', {'alt': '', 'class': 'mzp-c-billboard-image'}) }}
```

Comment:
This screenshot is from Developer Edition, shown in the box for Strict mode. Nightly doesn't seem to mention the browser by name so maybe that's changed.

Reply:
whoops! Looks like I missed one. I will retake this screenshot in Fx67.